tests/fixtures/fake_mq.py

```diff
@@ -90,7 +90,7 @@ def send_and_wait(self, topic: Text, message: ByteString, avoid_iteration_contro
     @gen.coroutine
     def getmany(self, avoid_iteration_control=False) -> Dict:
         # At the moment, this is the only topic we consume from ...
-        topic = "uploadvalidation"
+        topic = "platform.upload.validation"
```

Comment:
Wouldn’t be possible to pull this to some constant, either directly to ___init__.py___ or in a form of a pyTest fixture to _conftest.py_? Or even use the [constant](https://github.com/RedHatInsights/insights-upload/pull/48/files#diff-3f41e546893dc64b71aaacad12cad815R57) from the _app.py_. But again, it depends whether you are a fan of sharing code between the realms.